src/components/home/BillableHoursCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import Card from 'react-bootstrap/Card';
+import { Tooltip, OverlayTrigger, Container } from 'react-bootstrap';
+
+
+const renderTooltip = (props, updatedAt) => (
+  <Tooltip id="tooltip" {...props}>
+    {updatedAt}
+  </Tooltip>
+);
+export const LineItem = ({ label, value } : {label: string, value: string | number}) => (
+  <div className="TimeTracker-Hours-details">
+    <span>
+      {label}
+      :
+    </span>
```

Comment:
Did the formatter do this automatically? Just seems like a lot of line breaks

Reply:
Yes good catch - the linter thought it was multiple expressions, fixed it and disabled the "one expression per line" rule for that file so someone doesn't apply all rules and switch it back.